packages/cyberstorm/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
```

Comment:
Is this required?

Reply:
I think so, because of the usage of Radix. I'll try without it, while fixing #908

Reply:
Yes, tested.